taskt/Core/Automation/Commands/File Operation Commands/GetFilesCommand.cs

```diff
@@ -1,86 +1,73 @@
 
         public GetFilesCommand()
         {
-            this.CommandName = "GetFilesCommand";
-            this.SelectionName = "Get Files";
-            this.CommandEnabled = true;
-            this.CustomRendering = true;
+            CommandName = "GetFilesCommand";
+            SelectionName = "Get Files";
+            CommandEnabled = true;
+            CustomRendering = true;
         }
 
         public override void RunCommand(object sender)
         {
-            var engine = (Core.Automation.Engine.AutomationEngineInstance)sender;
+            var engine = (AutomationEngineInstance)sender;
             //apply variable logic
             var sourceFolder = v_SourceFolderPath.ConvertToUserVariable(sender);
 
-            //delete folder
-            //System.IO.Directory.Delete(sourceFolder, true);
-            var filesList = System.IO.Directory.GetFiles(sourceFolder).ToList();
-
-            Script.ScriptVariable newFilesList = new Script.ScriptVariable
-            {
-                VariableName = v_UserVariableName,
-                VariableValue = filesList
-            };
-            //Overwrites variable if it already exists
-            if (engine.VariableList.Exists(x => x.VariableName == newFilesList.VariableName))
-            {
-                Script.ScriptVariable temp = engine.VariableList.Where(x => x.VariableName == newFilesList.VariableName).FirstOrDefault();
-                engine.VariableList.Remove(temp);
-            }
-            engine.VariableList.Add(newFilesList);
+            //Get File Paths from the folder
+            var filesList = Directory.GetFiles(sourceFolder).ToList();
 
+            //Add File Paths to the output variable
+            engine.AddVariable(v_OutputUserVariableName, filesList);
         }
+
         public override List<Control> Render(frmCommandEditor editor)
         {
             base.Render(editor);
 
             RenderedControls.AddRange(CommandControls.CreateDefaultInputGroupFor("v_SourceFolderPath", this, editor));
-
-            RenderedControls.Add(CommandControls.CreateDefaultLabelFor("v_UserVariableName", this));
-            var VariableNameControl = CommandControls.CreateStandardComboboxFor("v_UserVariableName", this).AddVariableNames(editor);
-            RenderedControls.AddRange(CommandControls.CreateUIHelpersFor("v_UserVariableName", this, new Control[] { VariableNameControl }, editor));
-            RenderedControls.Add(VariableNameControl);
-
+            RenderedControls.AddRange(CommandControls.CreateDefaultOutputGroupFor("v_OutputUserVariableName", this, editor));
             return RenderedControls;
         }
+
         public override string GetDisplayValue()
         {
-            return base.GetDisplayValue() + " [From: '" + v_SourceFolderPath + "', Store In: '" + v_UserVariableName + "']";
+            return base.GetDisplayValue() + $" [From Folder '{v_SourceFolderPath}' - Store File Paths in '{v_OutputUserVariableName}']";
```

Comment:
"Paths" should be "Path(s)", but also, this gets stored as a list/array of strings right? Maybe the type should be "File Path(s) List". Please make these changes in File Extraction as well

Reply:
done